src/D2L.SQL.Validation/Language/SqlGrammar.cs

```diff
@@ -72,13 +69,12 @@ public SqlGrammar()
 			#endregion
 
 			#region Non-terminals
-			var Id = new NonTerminal( "Id" );
+			var TableAliasOpt = new NonTerminal( "IdOpt" );

```

Comment:
For consistency, can we change this to camelCase, and set the string value to the same name?